src/main/kotlin/com/dansplugins/factionsystem/lang/Language.kt

```diff
@@ -15,11 +15,32 @@ class Language(plugin: MedievalFactions, private val language: String) {
         val languageFolder = File(plugin.dataFolder, "lang")
         if (!languageFolder.exists()) {
             languageFolder.mkdirs()
-            plugin.saveResource("lang/lang_en_US.properties", false)
-            plugin.saveResource("lang/lang_en_GB.properties", false)
-            plugin.saveResource("lang/lang_fr_FR.properties", false)
-            plugin.saveResource("lang/lang_de_DE.properties", false)
         }
+
+        val en_us_filename = "lang/lang_en_US.properties"
```

Comment:
You shouldn't have underscores in variable names - does this pass linting?

Reply:
I believe this did pass linting. I can correct this.

Reply:
I've removed these underscores.